docs/docsite/rst/dev_guide/testing/sanity/no-underscore-variable.rst

```diff
@@ -0,0 +1,28 @@
+Sanity Tests » no-underscore-variable
+=====================================
+
+In the future, Ansible may use the identifier ``_`` to internationalize its
+message strings.  To be ready for that, we need to make sure that there are
+no conflicting identifiers defined in the code base.
+
+In common practice, ``_`` is frequently used as a dummy variable (a variable
+to receive a value from a function where the value is useless and never used).
+In Ansible, we're using the identifier ``dummy`` for this purpose instead.
```

Comment:
Looks like flake8 (interestingly, seems like flake8 only; pep8 and pyflakes don't complain) is not treating any dummy/unused value specially here.  I tried ```dummy, _dummy, __, and _``` and all of them result in F182 with flake8.